### DISCUSSION TITLE
Does Teslamate Need to keep running?

### DISCUSSION THREAD

**Discussion by mathewparet** [teslamate-org/teslamate#3160] [Q&A]
Does Teslamate need to keep running always? Or does it sync previous data when brought up?

At this point, I am only interested in tracking my electricity usage at various places.

So if I shutdown the docker instances and bring them back up only when I need to check the rates, will it sync previous data from Tesla and calculate the rates? Or will it always be up and running for this to work?

**Comment by wooter** [comment]
Tesla doesn't share any historic information on your car. Therefore, the only way to make a history of what your car is doing, is to regularly poll the Tesla API to see what your car is doing at that moment, and retain that information for historic purposes.

TL;DR: yes, TeslaMate needs to continually run to poll your car.
